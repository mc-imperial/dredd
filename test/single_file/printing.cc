#include <iostream>

int main() {
  std::cout << "A\n";
  std::cout << "B\n";
  std::cout << "C\n";
  std::cout << "D\n";
  std::cout << "E\n";
  std::cout << "F\n";
  std::cout << "G\n";
}
