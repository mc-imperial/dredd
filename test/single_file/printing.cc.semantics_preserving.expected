#include <iostream>

static unsigned long long int no_op = 0;

#include <cinttypes>
#include <cstddef>
#include <functional>
#include <string>


#ifdef _MSC_VER
#define thread_local __declspec(thread)
#elif __APPLE__
#define thread_local __thread
#endif

#include <limits>
#include <cmath>
#define MUTATION_RETURN(arg) actual_result
#define MUTATION_PRELUDE(arg,type) type actual_result = (arg)
int main() {
  std::cout << "A\n";
  std::cout << "B\n";
  std::cout << "C\n";
  std::cout << "D\n";
  std::cout << "E\n";
  std::cout << "F\n";
  std::cout << "G\n";
}
